packages/core-tabs/core-tabs.js

```diff
@@ -45,31 +101,39 @@ addEvent(UUID, 'keydown', (event) => {
   }
 })
 
-function getOpenPanel (panels) {
-  return Math.max(0, panels.indexOf(panels.filter((pan) => !pan.hasAttribute('hidden'))[0]))
+function getOpenTabIndex (tabs) {
+  return Math.max(0, tabs.indexOf(tabs.filter(tab => tab.getAttribute('aria-selected') === true)))
 }
 
-function setOpen (tablist, open) { // open can be Number, String or Element
-  const next = tablist.nextElementSibling.children
+/**
+ * @param {Element} tablist
+ * @param {Number|String|Element} open
+ * @param {Options} [options]
+ */
+function setOpen (tablist, open, options = {}) { // open can be Number, String or Element
+  const next = tablist.nextElementSibling ? tablist.nextElementSibling.children : []
   const tabs = queryAll(tablist.children).filter((tab) => tab.nodeName === 'A' || tab.nodeName === 'BUTTON')
   const panels = tabs.map((tab, i) => document.getElementById(tab.getAttribute('aria-controls')) || next[i])
-  const isOpen = getOpenPanel(panels)
+  const isOpen = getOpenTabIndex(tabs)
   const willOpen = tabs.reduce((acc, tab, i) => (i === open || tab === open || tab.id === open) ? i : acc, isOpen)
   const isUpdate = isOpen === willOpen || dispatchEvent(tablist, 'tabs.toggle', {isOpen, willOpen, tabs, panels})
-  const nextOpen = isUpdate ? willOpen : getOpenPanel(panels) // dispatchEvent can change attributes, so check getOpenPanel again
+  const nextOpen = isUpdate ? willOpen : getOpenTabIndex(tabs) // dispatchEvent can change attributes, so check getOpenPanel again
 
   tabs.forEach((tab, index) => {
     const selected = index === nextOpen
     const panel = panels[index]
```

Comment:
Tror kanskje jeg trenger at panels er et map, for rekkefølge fører til at jeg må flytte ludo:
```ts
 {[tabId: string]: Element}
```

Reply:
Nei, trenger ikke det. Mapper jo over tabs, og henter rett panel basert på ID.